.eslintrc

```diff
@@ -65,7 +65,6 @@
       "no-unreachable": 1,
       "no-unused-expressions": 1,
       "react/prefer-es6-class": 1,
-      "react/prop-types": 0,
-      "no-console": "off"
+      "react/prop-types": 0
```

Comment:
I get ESlint errors w/o `no-console: off`. There are a few `console.log` statements in `server/index.js`.